Homework/homework1/task4.py

```diff
@@ -0,0 +1,35 @@
+"""
+Classic task, a kind of walnut for you.
+
+Given four lists A, B, C, D of integer values,
+    compute how many tuples (i, j, k, l) there are such that A[i] + B[j] + C[k] + D[l] is zero.
+We guarantee, that all A, B, C, D have same length of N where 0 ≤ N ≤ 1000.
+"""
+from typing import List, Mapping
+
+
+def create_dict(x: int, x_dict: Mapping[int, int]) -> Mapping[int, int]:
```

Comment:
Simple notice if the function name and first verb in the docstring are not the same it is 90% a bug.